src/app/cli/src/dune

```diff
@@ -9,7 +9,7 @@
  (libraries mina_cli_entrypoint)
  (instrumentation (backend bisect_ppx))
  (preprocess (pps ppx_version))
- (flags -short-paths -g -w @a-4-29-40-41-42-44-45-48-58-59-60-66))
+ (flags -short-paths -w @a-4-29-40-41-42-44-45-48-58-59-60-66))
```

Comment:
Can you check with @mrmr1993 if we need the `-g` for the core-dumps or other data for debugging the memory errors?